services/graphql-server/src/graphql/definitions/website/schedule.js

```diff
@@ -98,6 +99,11 @@ input WebsiteScheduleOptionInput {
   status: ModelStatus = active
 }
 
+input CreateWebsiteSchedulesMutationInput {
+  contentId: Int!
```

Comment:
Change to `definitions: [CreateWebsiteScheduleDefinitionInput!]!`

and then do something like
```graphql
type CreateWebsiteScheduleDefinitionInput {
  contentId: Int!
  sectionId: Int!
  optionId: Int
  startDate: Date
  endDate: Date
}
```